java/rocksjni/abstract_associative_merge_operator.cc

```diff
@@ -0,0 +1,199 @@
+/*
+ * @author cristian.lorenzetto@gmail.com
+ * */
+
+
+#include <iostream>
+#include <jni.h>
+#include <rocksjni/init.h>
+#include <assert.h>
+#include <string>
+#include <memory>
+#include "util/logging.h"
+#include "rocksdb/slice.h"
+#include "include/org_rocksdb_AbstractAssociativeMergeOperator.h"
+#include "rocksjni/portal.h"
+#include "rocksjni/init.h"
+#include "rocksdb/merge_operator.h"
+#include "utilities/merge_operators.h"
+
+
+
+namespace rocksdb {
```

Comment:
Tabs and spacing don't look aligned with rest of codebase in all these .cc files